src/glmfit.jl

```diff
@@ -115,7 +115,23 @@ function confint(obj::AbstractGLM, level::Real)
 end
 confint(obj::AbstractGLM) = confint(obj, 0.95)
 
-deviance(m::AbstractGLM)  = deviance(m.rr)
+deviance(m::AbstractGLM) = deviance(m.rr)
+
+function loglikelihood(m::AbstractGLM)
```

Comment:
> Actually, the deviance is also computed by going over all observations.

Yes, but the computation of the deviance is just summing the contributions. Furthermore, since this version is also computing the deviance the observations are traversed twice.

> The issue is, as I mention in the second point of JuliaStats/StatsBase.jl#146

Sorry, I missed that. I'm able to match the exact values for `Binomial(n=1)` (which has loglik(y,y)=0) and `Poisson`, but haven't tried others. However, in general, it doesn't seem cheaper to compute `loglik(y,y)` than `loglik(mu,y)` because the formulas don't seem to simplify. That is why I suggested computing the log-likelihood as `-deviance/2`. It might confuse some that it's different from the joint logpdf, but it's sound, much cheaper, and can easily be documented.